contents/dataframes_indexing.md

```diff
@@ -147,15 +150,15 @@ filter(:name => ==("Alice"), grades_2020())
 """; process=without_caption_label)
 ```
 
-To get all the rows which are *not* Alice, `==` can be replaced by `!=` in all previous examples:
+To get all the rows which are *not* Alice, `==` (equality) can be replaced by `!=` (inequality) in all previous examples:
 
 ```jl
 s = """filter(:name => !=("Alice"), grades_2020())"""
 sco(s; process=without_caption_label)
 ```
 
-Now, to show why anonymous functions are so powerful, we can come up with a more complex filter.
-In this filter, we want to have the people whos name start with A or B **and** who have a grade above a 6:
+Now, to show **why anonymous functions are so powerful**, we can come up with a slight more complex filter.
+In this filter, we want to have the people whose name start with A or B **and** while also having grades over 6:
```

Comment:
```suggestion
In this filter, we want to have the people whose name start with A or B **and**  have a grade above 6:
```
Not like this? The "while" sounds off: "Select the people where A and B" versus "Select the people where A and while B".